src/test/scala/br/unb/cic/tip/parser/StatementParserTest.scala

```diff
@@ -0,0 +1,248 @@
+package br.unb.cic.tip.parser
+
+import org.scalatest.funsuite.AnyFunSuite
+import br.unb.cic.tip.*
+
+import br.unb.cic.tip.Stmt.*
+import br.unb.cic.tip.Expression.*
+
+class StatementParserTest extends AnyFunSuite {
+  ignore("should parse empty strings") {
+    // TODO: this is required by the grammar rules.
+    // Is it implementable  with the current AST?
+    val result = StatementParser.parse("")
```

Comment:
Good point. I think the AST does not support the empty statement.

Reply:
Done! Added a NopStmt to Syntax